TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Eel/FlowQueryOperations/FilterOperation.php

```diff
@@ -129,7 +129,17 @@ protected function evaluateOperator($value, $operator, $operand) {
 				$isOfType = $value->getNodeType()->isOfType($operand[0] === '!' ? substr($operand, 1) : $operand);
 				return $operand[0] === '!' ? $isOfType === FALSE : $isOfType;
 			}
+		} else if ($operator === '!instanceof' && $value instanceof \TYPO3\TYPO3CR\Domain\Model\NodeInterface) {
```

Comment:
Can't we just call `!$this->evaluateOperator($value, 'instanceof', $operand)` ? Looks less error-prone to me. What do you think? :-)